### ISSUE TITLE
False positives in list and dict comprehensions

### ISSUE THREAD

**Issue by danpalmer** [zedlander/flake8-commas#3]
I've just run this against our codebase, and it looks like most of the 'violations' are in fact list comprehensions that are split over multiple lines. For example:

```python
foo = [
    bar(x) for x in y
]
```

We do this to break up line length, and often even do this:

```python
foo = [
    bar(x)
    for x in y
]
```

**Edit:** the same happens for dict comprehensions as well.
